tests/groups.rs

```diff
@@ -18,7 +18,12 @@ USAGE:
 
 For more information try --help";
 
-static REQ_GROUP_CONFLICT_REV: &str = "error: The argument '--delete' cannot be used with '<base>'
+// FIXME: This message has regressed after https://github.com/clap-rs/clap/pull/1856
+//        Need to roll back somehow.
+static REQ_GROUP_CONFLICT_REV: &str =
```

Comment:
Could you add a test for usage string that has 2 conflicting options?

Reply:
"usage string that has 2 conflicting options"? How a string can have conflicting options 😄? Could you say few more words please?

Reply:
```
fn req_group_with_conflict_usage_string() {
    let app = App::new("req_group")
        .arg(Arg::from("-a, -all 'All'").conflicts_with("delete"))
        .arg(Arg::from(
            "-d, --delete 'Remove the base commit information'",
        ))
        .group(
            ArgGroup::with_name("all_or_delete")
                .args(&["all", "delete"])
                .required(true),
        );
    assert!(utils::compare_output2(
        app,
        "clap-test --delete --all",
        REQ_GROUP_CONFLICT_REV,
        REQ_GROUP_CONFLICT_USAGE,
        true
    ));
}
```

Reply:
That'll do.

Reply:
This message has not degraded because it's the degraded one already :(

Reply:
Wait a minute, @pksunkara shouldn't the `.arg(Arg::from("-a, -all 'All'").conflicts_with("delete"))` be `.arg(Arg::from("-a, --all 'All'").conflicts_with("delete"))`?

Reply:
Yeah, looks like a mistake.